tensorflow/go/tensor.go

```diff
@@ -34,6 +34,9 @@ import (
 // DataType holds the type for a scalar value.  E.g., one slot in a tensor.
 type DataType C.TF_DataType
 
+// Same as strconv.IntSize or math/bits.UintSize
+const intSize = 32 << (^uint(0) >> 32 & 1)
```

Comment:
Instead of intSize, could we use something like `unsafe.Sizeof(unsafe.Pointer(nil)) == 8`?

Reply:
Done.

Do you want me to extract the conversion into a utility function in tensor.go instead of duplicating in two places?